templates/404/invalid_huc.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'base.html' %}
+{% block content %}
+<h3>Invalid HUC-8</h3>
+<p>
+  The HUC-8 code you provided is not supported.
```

Comment:
Text has been changed to "invalid" in c1b774a. My vote is to leave the contact info off the error pages since it'd be one more place we would need to maintain contact information.